optimade/models/jsonapi.py

```diff
@@ -137,8 +137,11 @@ class BaseResource(BaseModel):
 
 
 class RelationshipLinks(BaseModel):
-    """A resource object **MAY** contain references to other resource objects (\"relationships\").
-    Relationships may be to-one or to-many. Relationships can be specified by including a member in a resource's links object."""
+    """ A resource object **MAY** contain references to other resource objects (\"relationships\").
+    Relationships may be to-one or to-many.
+    Relationships can be specified by including a member in a resource's links object.
+
+    """
 
     self: Optional[Union[AnyUrl, Link]] = Field(None, description="A link to itself")
     related: Optional[Union[AnyUrl, Link]] = Field(
```

Comment:
The description here should be:
`"A [related resource link](https://jsonapi.org/format/1.0/#document-resource-object-related-resource-links)"`.

Reply:
Replaced the entire docstring with this

Reply:
I guess for `related`'s OpenAPI description you mean, right?

Reply:
Errr, I do now...